### PR TITLE
Turkish flag poster added (Extremely vital for game balance)

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
Adds a Turkish flag poster into the game (doesn't place it anywhere however)

## Why It's Good For The Game
The Brits get their funny coffee mugs and underwear options. This extreme amount of British influence cannot stand so I propose adding this poster to balance this injustice. 

Also this: 
image

THIS PR IS EXTREMELY VITAL TO THE DEVELOPMENT OF TUBESTATION

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

image

Here is a group of extremely based posters placed around a room for comedic affect. 

</details>

## Changelog
:cl: Phil Smith
add: Turkish flag poster now exists in the code :trollface: 
/:cl:

### CODE REVIEW

**Comment by AnCopper** [comment]
image
Turkish funeral.

**Comment by TeomanTheGreat** [comment]
> image
> Turkish funeral.

IS THAT A CROSS ON THAT CASKET? smh

**Comment by BNGP0** [comment]
At the one hand turning beestation development into "add a random item from your country because someone may forget it exists" may slow down development of updates which add more interesting mechanics to the game, at the other hand i want to add Adidas hardsuits so i support it

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -662,4 +662,9 @@
 	desc = "A Syndicate-commissioned poster that uses Syndie Moth(TM?) to tell the viewer to keep the nuclear authentication disk unsecured. No, we aren't doing that. It's signed by 'AspEv'."
 	icon_state = "poster49"
 
+/obj/structure/sign/poster/contraband/turkiye // extremely essential to counteract the imperialist forces of the British mug which is also in the game
+	name = "Turkish Flag"
+	desc = "The flag of the former nationstate of Turkey. How did this get on the station?"
```
Comment (Archanial) [inline code/game/objects/effects/contraband.dm]:
```suggestion
	desc = "when allah created world allah did give whole world to turkey but turkey frendly countrie so turkey gived land to other countrie"
```

Comment (TeomanTheGreat) [reply code/game/objects/effects/contraband.dm]: I shall resolve this when i get home

**Comment by Crossedfall** [comment]
Funny, but no.

**Comment by TeomanTheGreat** [comment]
image

**Comment by super12pl** [comment]
Greek admins

**Pull request closed without merging**